openssl/src/crypto/sign.rs

```diff
@@ -0,0 +1,274 @@
+use ffi;
```

Comment:
Might be nice to consider adding back something like the examples in the docs from #420: https://github.com/sfackler/rust-openssl/pull/420/files#diff-35c3c6c147bd45b93218a395c7901036R3